src/SignalRServiceExtension/SignalRAttribute.cs

```diff
@@ -10,10 +10,8 @@ namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
     [Binding]
     public class SignalRAttribute : Attribute
     {
+        public string ConnectionStringSetting { get; set; } = Constants.AzureSignalRConnectionStringName;
 
-        [AppSetting(Default = Constants.AzureSignalRConnectionStringName)]
```

Comment:
why remove `AppSetting`

Reply:
For multiple endpoints scenarios, we have to resolve multiple connection strings with our customized method, which requires the actual value of `ConnectionStringSetting` set by users served as the section name of configuration.

Reply:
```
    /// Place this on binding attributes properties to tell the binders that that the property
    /// should be automatically resolved as an app setting
```
https://github.com/Azure/azure-webjobs-sdk/blob/72cf608c3891045390b0f26d7ecec63c8a81cef7/src/Microsoft.Azure.WebJobs/AppSettingAttribute.cs#L9
Looks like we can still use it according to its description. 

```
// then app setting
            if (appSettingAttr != null)
            {
                return GetConfigurationResolver(str, appSettingAttr.Default, propInfo, validator, s => _configuration[s]);
            }
```
https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs#L133
looks like add the <connStrKey, connStrs> in IConfiguration is fine?

Reply:
The `ConfigurationResolver` only resolves strings, but we want `ServiceEndpoint[]`. What's more, this seems to be not dynamic.

Reply:
what's your plan? can we use this one? https://docs.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-azure-functions-csharp#reload-data-from-app-configuration to make the configuration refreshable?

convert resolved string to ServiceENdpoint[] ?